CONTRIBUTING.md

```diff
@@ -10,6 +10,33 @@ Use HashiCorp's [Plugin Development](https://www.terraform.io/plugin) guide as a
 
 See the [Developing The Provider](#developing-the-provider) section below for specifics about this GitLab provider.
 
+### GitPod Environment Setup
```

Comment:
Please move this section to be underneath the "Developing The Provider" heading. This portion of the document is intended as an overview.

Reply:
This section fits as an "option" heading underneath "Running Tests"

Reply:
hey @armsnyder  - just so I understand, are you suggesting move the whole "GitPod Environment Setup" to an "Option 3" under the Running Tests header? Or are you suggesting that we move "GitPod Environment Setup" into a separate sub-header under "Developing The Provider", then add an "Option 3" for "Use GitPod" under running tests.

I think it makes sense to move the header under "Developing the provider", but I don't know if I'd want to move the whole header to underneath the tests, since the use of GitPod is more than just running the tests.

Reply:
I see. Yes in that case it can be a heading under "Developing The Provider" for remote-style development.

Reply:
Perfect, I'll get that moved. Thanks!